### ISSUE TITLE
CentOS 7 - kubelet package requirements iproute2 and mount?

### ISSUE THREAD

**Issue by catalinpan** [kubernetes/release#204]
Hi,

I have tried to build the deployment rpms from master branch but on CentOS 7 I cannot install the packages as the dependecies are not found.
The changes have been added on below commit

https://github.com/kubernetes/release/commit/7c9fb0f4445b692f2466b51af0bca0908c800cf8#diff-c8126c3004cef03f9bb0d3352adb80a8R27

yum provides */iproute2    shows **iproute** as package which containes iproute2
and
yum provides */mount    shows **util-linux** as package which containes mount

Should I do a PR or is there any alternative?

Thanks

**Comment by pennpeng** [comment]
I met the same problem!!!!!
"-> Processing Dependency: mount for package: kubelet-1.4.4-2.x86_64
--> Processing Dependency: iproute2 for package: kubelet-1.4.4-2.x86_64
---> Package socat.x86_64 0:1.7.2.2-5.el7 will be installed
--> Finished Dependency Resolution
Error: Package: kubelet-1.4.4-2.x86_64 (kubernetes)
           Requires: iproute2
Error: Package: kubelet-1.4.4-2.x86_64 (kubernetes)
           Requires: mount
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
"

**Comment by resouer** [comment]
@luxas Could you please take a look?

**Comment by ixdy** [comment]
fix merged into kubernetes/release; I'm not sure what needs to be done to get this published.
